howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,85 @@
+The Anbox Cloud Appliance uses the self-signed certificate to provide HTTPS services. If you want to serve the appliance over HTTPS using a valid SSL/TLS certificate, this document will walk you through the steps in generating and installing a valid SSL/TLS certificate on Anbox Cloud Appliance.
```

Comment:
```suggestion
The Anbox Cloud Appliance uses a self-signed certificate to provide HTTPS services. If you want to serve the appliance over HTTPS using a valid SSL/TLS certificate, follow the steps in this document to generate and install a valid SSL/TLS certificate on the Anbox Cloud Appliance.
```

(The reason for this change: The document will walk you through the steps no matter if you want to serve the appliance over HTTPS or not. :smile: )

Reply:
Ah, fair enough.